src/verifact_manager_openrouter.py

```
@@ -0,0 +1,313 @@
+class ManagerConfig(BaseModel):
+    """Configuration options for the factcheck pipeline."""
+    min_checkworthiness: float = Field(0.5, ge=0.0, le=1.0)
+    max_claims: Optional[int] = None
+    evidence_per_claim: int = Field(5, ge=1)
+    timeout_seconds: float = 120.0
+    enable_fallbacks: bool = True
+    retry_attempts: int = 2
+    raise_exceptions: bool = False
+    include_debug_info: bool = False
+
+class VerifactManager:
```

Ruff (0.11.9)

Missing docstring in public class

(D101)